test/fakeintake/server/server.go

```diff
@@ -58,10 +58,10 @@ func NewServer(options ...func(*Server)) *Server {
 
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", fi.handleDatadogRequest)
-	mux.HandleFunc("/fakeintake/payloads/", fi.handleGetPayloads)
-	mux.HandleFunc("/fakeintake/health/", fi.handleFakeHealth)
-	mux.HandleFunc("/fakeintake/routestats/", fi.handleGetRouteStats)
-	mux.HandleFunc("/fakeintake/flushPayloads/", fi.handleFlushPayloads)
+	mux.HandleFunc("/fakeintake/payloads", fi.handleGetPayloads)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
TIL 2